Tensile/Common.py

```diff
@@ -1223,9 +1223,9 @@ def getArchitectureName(gfxName):
     # Typically matching 16 bytes is good choice since the stores will be optimally coalesced with 16 bytes/WI.
     # -1 means use the largest vector width up to 128 bits.
     # Using a VW too large which results in >16bytes/thread isn't supported
-    # For MFMA non SourceSwap: this parameter didn't take effect
-    # For MFMA SourceSwap: this parameter only take effect on A buffer for now
     "VectorWidth":                [ -1, 1, 2, 3, 4, 6, 8, 16 ],
+    # VectorWidth for B (MatrixInstruction only)
+    "VectorWidthB":               [ -1, 1, 2, 4, 8, 16 ],
```

Comment:
Does this refer to VectorWidth for B matrix or for block dimension of matrix instruction?

Reply:
Looks like matrix B, so "VectorWidth" now becomes "VectorWidthA"?

Reply:
> Looks like matrix B, so "VectorWidth" now becomes "VectorWidthA"?

Internally, I added kernel["VectorWidthA"] (the value is same as "VectorWidth" ).
It is very difficult to change "VectorWidth" to "VectorWidthA".
I need to change all existing test cases and logic yaml files....

Reply:
As I put a comment here, VectorWidthB is for MatrixInstruction only.
We still use VectorWidth for non MI part.
Of course we can change them to VectorWidthA. This is just to make sure we did not change the logic for non MI part.